src/tools/illink/test/Mono.Linker.Tests.Cases/Mono.Linker.Tests.Cases.csproj

```diff
@@ -68,7 +68,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="System.Security.Permissions" Version="5.0.0" />
+    <PackageReference Include="System.Security.Permissions" Version="$(SystemSecurityPermissionsVersion)" />
```

Comment:
It's unfortunate that we don't have a good way to establish a live dependency here without bringing in the entire libs build graph. You clearly stated that when building the linker, you want to avoid to build the "clr+libs" subsets which is why you chose to build against an LKG instead. But by using an LKG, we now bring in these OOB prebuilts that we can either add to dependency flow or just keep pinned to a static version.

Reply:
cc @ericstj

Reply:
Could we refactor these tests to be part of the libs test subset, or some test subset that *does* depend on both libs and linker?

Reply:
Sure, we can always sequence them very late or make them depend on the entire libs subset to be built but that's not what the linker team wants. They prefer the least number of pre-steps to build and test the linker, and depending on an LKG seems like the right compromise here.

Another thought, we could make sure that these tests depend on the `NetCoreAppPrevious` asset of these OOB libraries instead of `NetCoreAppCurrent` which would be achievable via a ProjectReference and a `SetTargetFramework=$(NetCoreAppPrevious)` metadata on it.